apphost/src/runtime.rs

```diff
@@ -163,4 +166,36 @@ pub fn launch_app_binary(runtime_dir: Option<&Path>, app_info: &AppInfo) -> Resu
         Ok(err) => Err(Box::new(err)),
         Err(_) => Ok(res)
     }
-}
\ No newline at end of file
+}
+
+
+#[cfg(unix)]
```

Comment:
Would be ideal if this could be moved into a separate module, i.e.
```rs
#[cfg(unix)]
mod unix {
    use ...;
    
    fn fixup_signal_handling() {
        ...
    } 
}
```